scripts/ldr

```diff
@@ -90,8 +90,10 @@ case $1 in
     "start")
       roachprod sql $A:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
       roachprod sql $B:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
-      roachprod sql $A:1 -- -e "CREATE EXTERNAL CONNECTION IF NOT EXISTS b AS $(roachprod pgurl --database ycsb --insecure $B:1)"
-      roachprod sql $B:1 -- -e "CREATE EXTERNAL CONNECTION IF NOT EXISTS a AS $(roachprod pgurl --database ycsb --insecure $A:1)"
+      a_conn="$(roachprod ssh $A:1 -- ./cockroach encode-uri --inline --ca-cert ./certs/ca.crt --key ./certs/client.root.key --cert ./certs/client.root.crt {pgurl:1} | sed 's,postgresql://postgres://,postgres://,' | sed 's/roachprod:cockroachdb/root/' | sed 's/defaultdb/ycsb/')"
+       b_conn="$(roachprod ssh $B:1 -- ./cockroach encode-uri --inline --ca-cert ./certs/ca.crt --key ./certs/client.root.key --cert ./certs/client.root.crt {pgurl:1} | sed 's,postgresql://postgres://,postgres://,' | sed 's/roachprod:cockroachdb/root/' | sed 's/defaultdb/ycsb/')"
```

Comment:
Perhaps we can use the  `--database` and `--user` flags to help avoid a couple of these sed commands? A comment on why first one is necessary would be good as well.  I also thought that @dt fixed the duplicate scheme printing bug.  Are we running old cockroach binaries with this script?

Reply:
arg sorry, i meant to close this out. adam lobbied me to keep using insecure clusters.